lib/streamlit/elements/widgets/button.py

```diff
@@ -421,6 +424,30 @@ def link_button(
             use_container_width=use_container_width,
         )
 
+    @gather_metrics("page_link")
+    def page_link(
+        self,
+        page: str,
+        *,
+        label: Optional[str] = None,
+        icon: Optional[str] = None,
+        help: Optional[str] = None,
+        disabled: bool = False,
+        use_container_width: Optional[bool] = None,
+    ) -> "DeltaGenerator":
+        """Display a link to another page in a multi-page app.
```

Comment:
nit: some docstring is missing (params & examples), will it be added by doc team in a later PR?